ledger/src/blockstore.rs

```diff
@@ -721,6 +724,7 @@ impl Blockstore {
         self.program_costs_cf.submit_rocksdb_cf_metrics();
         self.bank_hash_cf.submit_rocksdb_cf_metrics();
         self.optimistic_slots_cf.submit_rocksdb_cf_metrics();
+        self.merkle_root_meta_cf.submit_rocksdb_cf_metrics();
```

Comment:
I guess this is probably the line that prevents clippy from complaining about unused member?

Reply:
yep. If you'd prefer I can remove this line and add the `#[allow(dead_code)]` to the column

Reply:
Nah, let's leave it; more thinking out loud